db/listen.js

```diff
@@ -0,0 +1,4 @@
+const app = require('express')();
+const port = 8000;
+app.listen(port);
+console.log('Listening to: Localhost:8000...');
```

Comment:
we want this log to run when app successfully listens so where would be a more appropriate place to put this? At the moment due to where it is right now, it will always run, whether app listens successfully or not...